src/lib/public/board/data/officers.json

```diff
@@ -1,718 +1,368 @@
-        "tier": 20
-      },
-      "S22": {
-        "title": "Dev President",
-        "tier": 20
-      },
-      "F22": {
-        "title": "Marketing Co-Lead",
-        "tier": 23
-      }
-    },
-    "picture": "wesley-chou.webp"
+      "S21": { "title": "Dev Officer", "tier": 19 },
+      "F21": { "title": "Dev Director", "tier": 17 },
+      "S22": { "title": "Dev President", "tier": 17 },
+      "F22": { "title": "Marketing Co-Lead", "tier": 8 }
+    }
   },
   {
     "fullName": "Ethan Davidson",
+    "picture": "ethan-davidson.webp",
     "socials": {
-      "website": "ethandavidson.com",
       "github": "EthanThatOneKid",
       "discord": "EthanThatOneKid#3456",
-      "linkedin": "etok",
-      "instagram": "ethanthatonekid"
+      "linkedin": "etok"
     },
     "positions": {
-      "S21": {
-        "title": "Webmaster",
-        "tier": 5
-      },
-      "F21": {
-        "title": "Webmaster",
-        "tier": 5
-      },
-      "S22": {
-        "title": "Webmaster",
-        "tier": 5
-      },
-      "F22": {
-        "title": "Vice President/Webmaster",
-        "tier": 2
-      }
-    },
-    "picture": "ethan-davidson.webp"
+      "S21": { "title": "Webmaster", "tier": 2 },
+      "F21": { "title": "Webmaster", "tier": 2 },
+      "S22": { "title": "Webmaster", "tier": 2 },
+      "F22": { "title": "Vice President/Webmaster", "tier": 1 }
+    }
   },
   {
     "fullName": "Kevin Dillon",
-    "positions": {
-      "S21": {
-        "title": "Algo Officer",
-        "tier": 15
-      }
-    },
-    "picture": "kevin-dillon.webp"
+    "picture": "kevin-dillon.webp",
+    "positions": { "S21": { "title": "Algo Officer", "tier": 12 } }
   },
   {
     "fullName": "Diamond Dinh",
-    "socials": {
-      "github": "diamondburned"
-    },
+    "picture": "diamond-dinh.webp",
+    "socials": { "github": "diamondburned" },
     "positions": {
-      "S22": {
-        "title": "Dev Project Manager",
-        "tier": 22
-      },
-      "F22": {
-        "title": "Special Events Officer",
-        "tier": 25
-      }
-    },
-    "picture": "diamond-dinh.webp"
+      "S22": { "title": "Dev Project Manager", "tier": 18 },
+      "F22": { "title": "Special Events Officer", "tier": 7 }
+    }
   },
   {
     "fullName": "Eduardo Gomez",
-    "positions": {
-      "S21": {
-        "title": "nodebuds Officer",
-        "tier": 23
-      }
-    },
-    "picture": "eduardo-gomez.webp"
+    "picture": "eduardo-gomez.webp",
+    "positions": { "S21": { "title": "nodebuds Officer", "tier": 22 } }
   },
   {
     "fullName": "Nurhaliza Hassan",
+    "picture": "nurhaliza-hassan.webp",
     "positions": {
-      "S22": {
-        "title": "Marketing Director",
-        "tier": 12
-      },
-      "F22": {
-        "title": "Marketing Co-Lead",
-        "tier": 23
-      }
-    },
-    "picture": "nurhaliza-hassan.webp"
+      "S22": { "title": "Marketing Director", "tier": 8 },
+      "F22": { "title": "Marketing Co-Lead", "tier": 8 }
```

Comment:
The updated "tier" for Marketing Leader is '9' (both lines 93 and 94)

`"Marketing Leader": { "id": 9, "index": 550 }`